fdpa-resolutions.html

```diff
@@ -2,79 +2,21 @@
 <html>
   <head>
   <title>Fair Districts PA - Local Resolutions</title>
+  <meta name="referrer" content="origin">

```

Comment:
Necessary for the rawgit references below.

Reply:
Seems like an HTML comment is needed here. Something like
```
<!-- Necessary for rawgit and google fusiontables cross origin requests -->
```
You agree, @amyprime?

Reply:
Yeah, and adding the fusiontables to the comment makes it even better.